server/controllers/userController.js

```diff
@@ -0,0 +1,82 @@
+const { User } = require('../models/index');
+
+const getAllUsers = async (req, res) => {
+    try {
+        const users = await User.findAll();
+        if (!users) {
+            res.status(404).send("Error fetching users: Users not found");
+        } else {
+            res.status(200).json(users);
+        }
+    } catch (error) {
+        res.status(500).send(error);
+    }
+}
+
+const getUser = async (req, res) => {
+    try {
+        const userId = req.params.userId
+        const user = await User.findByPk(userId);
+        if (!user) {
+            res.status(404).send(`Error fetching user: User with ID ${userId} not found`);
+        } else {
+            res.status(200).json(user);
+        }
+    } catch (error) {
+        res.status(500).send(error);
+    }
+}
+
+const createUser = async (req, res) => {
+    try {
+        const userInfo = req.body;
+        await User.create(userInfo);
+        const newUser = await User.findOne({ where: { email: userInfo.email } });
+        if (!newUser) {
+            res.status(500).json({ error: 'Error creating user '});
```

Comment:
i'd send back 400 here instead of 500, 400 signals 'bad request' which is more suitable than 500 'internal server error'